packages/service/src/rehearsal-service-host.ts

```diff
@@ -32,9 +33,17 @@ export class RehearsalServiceHost implements LanguageServiceHost {
     this.fileNames = fileNames;
   }
 
+  useCaseSensitiveFileNames = (): boolean => {
```

Comment:
Compiler resolves paths based on this function result :)

Reply:
This was a weird issue.